x-pack/plugins/apm/common/rules/schema.ts

```diff
@@ -9,6 +9,16 @@ import { schema, TypeOf } from '@kbn/config-schema';
 import { ML_ANOMALY_SEVERITY } from '@kbn/ml-anomaly-utils/anomaly_severity';
 import { AggregationType, ApmRuleType } from './apm_rule_types';
 
+export const searchConfigurationSchema = schema.object({
+  query: schema.object({
+    query: schema.oneOf([
+      schema.string(),
+      schema.recordOf(schema.string(), schema.any()),
+    ]),
+    language: schema.string(),
+  }),
+});
+
 export const errorCountParamsSchema = schema.object({
```

Comment:
Can you do me a favour and rename `errorCountParamsSchema ` to  `errorCountParamsRt` (same with the other rule param types). That is the naming with use elsewhere for runtime types.